src/aalimagecapturecontrol.cpp

```diff
@@ -172,7 +179,13 @@ void AalImageCaptureControl::saveJpeg(const QByteArray& data)
 
     // Restart the viewfinder and notify that the camera is ready to capture again
     if (m_service->androidControl()) {
-        android_camera_start_preview(m_service->androidControl());
+        // SUPER HACK! completly destroy and recreate camera hal to restart preview
```

Comment:
The problem also occurs when flipping from front to rear camera or when entering video recording mode, so the workaround will have to happen there too.

Reply:
Will fix that